guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p>Are you saying "In the second stage of Mask R-CNN"? It's a bit confusing here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>No, I am talking about Faster RCNN only. In my understanding, MaskRCNN is made of Faster R-CNN + Mask Head where Faster R-CNN's second stage has a RoIAlign layer instaed of RoIPool.</p>